### PR TITLE
(vite) - Skip transforming node_modules

### PR DESCRIPTION
**Pull request by intrnl**
vite `1.0.0-beta.12` now resolves full path for transforms during dev.

### CODE REVIEW

**Comment by intrnl** [comment]
Whoops!

**Comment by intrnl** [comment]
Is that how I'm supposed to do it? :sweat_smile:

**Comment by JoviDeCroock** [comment]
Hey, that’s exactly how you should do it. Has vite dropped the at modules path?

**Comment by intrnl** [comment]
`@modules` path is still being used, however, transforms during dev uses the actual file path.

image

**Comment by JoviDeCroock** [comment]
Awesome, let's keep both though for backwards compatability to former beta's.

File: .changeset/hungry-rings-reflect.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@prefresh/vite': minor
```
Comment (JoviDeCroock) [inline .changeset/hungry-rings-reflect.md:2]:
```suggestion
'@prefresh/vite': patch
```

**Comment by JoviDeCroock** [comment]
@intrnl Thanks for the PR this has been published as 0.10.2

**Pull request merged**